packages/components/src/settings/feature-options-defaults.json

```diff
@@ -213,6 +213,36 @@
       "minValue": 1,
       "maxValue": 100,
       "values": []
+    },
+    "showBetaCodesInfo": {
+      "defaultValue": false,
+      "labelText": "Show Betacode to Unicode Mappings inside Lookup box",
```

Comment:
change labelText to "Show Betacode to Unicode Mapping Help"

Reply:
done